armi/reactor/converters/axialExpansionChanger.py

```diff
@@ -173,41 +172,27 @@ def _isTopDummyBlockPresent(self):
         blkLst = self.linked.a.getBlocks()
         if not blkLst[-1].hasFlags(Flags.DUMMY):
             runLog.warning(
-                "No dummy block present at the top of {0}! "
+                f"No dummy block present at the top of {self.linked.a}! "
                 "Top most block will be artificially chopped "
-                "to preserve assembly height".format(self.linked.a)
+                "to preserve assembly height"
             )
             if self._detailedAxialExpansion:
                 msg = "Cannot run detailedAxialExpansion without a dummy block at the top of the assembly!"
                 runLog.error(msg)
                 raise RuntimeError(msg)
 
-    def axiallyExpandAssembly(self, thermal: bool = False):
-        """Utilizes assembly linkage to do axial expansion
-
-        Parameters
-        ----------
-        thermal : bool, optional
-            boolean to determine whether or not expansion is thermal or non-thermal driven
-
-        Notes
-        -----
-            The "thermal" parameter plays a role as thermal expansion is relative to the
-            BOL heights where non-thermal is relative to the most recent height.
-        """
+    def axiallyExpandAssembly(self):
+        """Utilizes assembly linkage to do axial expansion"""
         mesh = [0.0]
         numOfBlocks = self.linked.a.countBlocksWithFlags()
         runLog.debug(
             "Printing component expansion information (growth percentage and 'target component')"
-            "for each block in assembly {0}.".format(self.linked.a)
+            f"for each block in assembly {self.linked.a}."
         )
         for ib, b in enumerate(self.linked.a):
 
-            runLog.debug(msg="  Block {0}".format(b))
-            if thermal:
-                blockHeight = b.p.heightBOL
-            else:
-                blockHeight = b.p.height
+            runLog.debug(msg=f"  Block {b}")
+            blockHeight = b.p.height
```

Comment:
I'm curious if `getHeight()` should be used here rather than accessing the parameter directly.

Reply:
We certainly can. The `getHeight` method just returns the height parameter straight up.
https://github.com/terrapower/armi/blob/59b0cad5da848f8dfd1fa65e6cb858f42f964061/armi/reactor/blocks.py#L456-L458

Reply:
Addressed. https://github.com/terrapower/armi/pull/1032/commits/a61b5b02579f87e405e2d401085f9c314fe65351